src/frontend/magnify/src/components/auth/SignupForm/SignupForm.tsx

```diff
@@ -51,8 +58,47 @@ const messages = defineMessages({
   },
 });
 
+interface SignupErrors {
+  current_password?: string[];
+  new_password?: string[];
+}
+export interface SignupFormData {
```

Comment:
Ditto, and here you used the keyword `data` instead of `values` iin the LoginForm